src/web/util/DeviceInfo.js

```diff
@@ -33,9 +33,13 @@ module.exports = Class(function () {
 
   this.getChromeSize = function (isRotated) {
     var opts = this._opts;
-    var size = opts && opts.background
-             ? new Size(opts.background.width, opts.background.height)
-             : new Size();
+    var bg = opts && opts.background;
+
+    if (Array.isArray(bg)) {
+      bg = bg[0];
+    }
+
+    var size = bg ? new Size(bg.width, bg.height) : new Size();
```

Comment:
We should just delete the rotated images.  They're legacy from when we didn't do browser transforms and canvas rendering.  Now that we render the phone chrome in canvas, it seems unnecessary to have a separate image.